CPU bursting info (#138)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/upgrade-path.adoc

= Upgrade Runtime Fabric Components
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> Anypoint Runtime Fabric contains a collection of services which run within the cluster, including the agent used to connect to Anypoint Platform. Perform the following steps to upgrade these services.

Reviewer: ```suggestion
Anypoint Runtime Fabric 1.3 and 1.4 contain a collection of services that run within the cluster, including the agent used to connect to Anypoint Platform. To upgrade these services, you must meet some prerequisites, understand deployed application availability, apply the upgrade, and perform troubleshooting if necessary.
```
Suggested rewording is an example of making sure that the container topic text introduces contained subtopics.

> The following instructions are applicable to Runtime Fabric Versions 1.3 and 1.4.

Reviewer: ```suggestion
```

== Before You Begin

> * Run the upgrade procedure on a non-production Runtime Fabric to verify compatibility with your environment.

Reviewer: ```suggestion
* Run the upgrade procedure on a nonproduction instance of Runtime Fabric to verify compatibility with your environment.
```

> * Plan the upgrade procedure when your team does not need to deploy new applications or managing existing applications on Runtime Fabric.

Reviewer: ```suggestion
* Plan to upgrade when your team is not deploying new applications or managing existing applications on Runtime Fabric.
```

> ** The upgrade should be complete within 10 minutes in most cases.

Reviewer: ```suggestion
** The upgrade takes fewer than 10 minutes in most cases.
```

> ** The Runtime Fabric and Anypoint Runtime Manager connection may disconnect during the upgrade procedure. Connectivity is restored after the upgrade completes.

Reviewer: ```suggestion
** If the Runtime Fabric and Anypoint Runtime Manager connection disconnects during the upgrade, connectivity is restored after the upgrade finishes.
```
See style ref. entry for "complete."

== Availability of Deployed Applications

> The upgrade procedure is designed to keep deployed applications running and available to serve requests. The majority of the services which are upgradable via this process run within the controller nodes. Impact to running Mules and the worker nodes are minimal by design.

Reviewer: ```suggestion
The upgrade procedure is designed to not interfere with deployed applications. The majority of the services that are upgradable via this process run within the controller nodes. Impact to running instances of Mule and the worker nodes are minimal by design.
```
See style ref. entry for "that vs. which"

== Apply the Component Upgrade

> . Navigate to your Runtime Fabric by selecting your Runtime Fabric on the Runtime Fabrics tab under Runtime Manager.

Reviewer: ```suggestion
. Navigate to your Runtime Fabric instance by selecting it on the Runtime Fabrics tab in Runtime Manager.
```

Author: Updates applied as needed.

> . If an upgrade is available, click on the "Upgrade to v1.x" button. The status of the Runtime Fabric will update to reflect the upgrade procedure is in progress.

Reviewer: ```suggestion
. If an upgrade is available, click "Upgrade to v1.x". 
The status of the Runtime Fabric updates to reflect that the upgrade procedure is in progress.
```

Author: Updates applied as needed.

> . Wait for the status to transition to Active.

Reviewer: ```suggestion
. Wait for the upgrade status to transition to Active.
```

Author: Updates applied as needed.

> . Once the status transitions to Active, and the version of the Runtime Fabric reflects the new version, the upgrade is complete.

Reviewer: This is not a step.
```suggestion
. After both the status transitions to Active and the version of Runtime Fabric reflects the new version, the upgrade is complete.
```

Author: Updates applied as needed.

> == Troubleshooting Upgrade Errors

Reviewer: ```suggestion
== Troubleshoot Upgrade Errors
```

Author: Updates applied as needed.
